app/src/main/java/com/google/samples/apps/sunflower/compose/plantdetail/PlantDetailView.kt

```diff
@@ -245,37 +249,44 @@ private fun PlantImage(
     modifier: Modifier = Modifier,
     placeholderColor: Color = MaterialTheme.colors.onSurface.copy(0.2f)
 ) {
-    CoilImage(
-        data = imageUrl,
-        contentScale = ContentScale.Crop,
-        fadeIn = true,
-        contentDescription = null,
-        loading = {
+    Box(
+        modifier = modifier
+            .fillMaxWidth()
+            .height(Dimens.PlantDetailAppBarHeight)
+    ) {
+        val painter = rememberCoilPainter(request = imageUrl, fadeIn = true)
+        Image(
+            painter = painter,
+            contentDescription = null,
+            contentScale = ContentScale.Crop,
+            modifier = Modifier.fillMaxSize(),
```

Comment:
Nit: should probably be matchParentSize() instead

Reply:
Done, thanks!